exampleSite/content/awesome-team/gator.md

```diff
@@ -10,7 +11,6 @@ selfLink = "member-fragment"
 title = ""
 subtitle = ""
 
-
 [[member]]
```

Comment:
[[member]] is not necessary anymore. That would add another layer, which is unnecessary.